python/mergeTreeClustering.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python
+
+from paraview.simple import *
+
+# create a new 'XML Image Data Reader'
+isabelvti = XMLImageDataReader(registrationName='isabel.vti', FileName=['isabel.vti'])
+isabelvti.PointArrayStatus = ['velocityMag_02', 'velocityMag_03', 'velocityMag_04', 'velocityMag_05', 'velocityMag_30', 'velocityMag_31', 'velocityMag_32', 'velocityMag_33', 'velocityMag_45', 'velocityMag_46', 'velocityMag_47', 'velocityMag_48']
```

Comment:
I don't believe this is needed for the script to work.

Reply:
Indeed! Done in the next commit!